lasagne/layers/helper.py

```diff
@@ -349,7 +357,8 @@ def get_all_params(layer, **tags):
     True
     """
     layers = get_all_layers(layer)
-    params = chain.from_iterable(l.get_params(**tags) for l in layers)
+    params = chain.from_iterable(l.get_params(
+                          unwrap_shared=unwrap_shared, **tags) for l in layers)
```

Comment:
We usually use 8 characters for indentation in this case. Would be good to keep this consistent.